app/Models/User.php

```diff
@@ -40,6 +40,13 @@ class User extends Authenticatable implements MustVerifyEmail, HasLocalePreferen
 
     public const MAPS_SITE_OPEN_STREET_MAPS = 'open_street_maps';
 
+    /**
+     * Possible distance unit.
+     */
+    public const DISTANCE_UNIT_MILES = 'miles';
```

Comment:
`mi` is the unit for miles.
```suggestion
    public const DISTANCE_UNIT_MILES = 'mi';
```